### PR TITLE
Enable YJIT after server boot

### PR DESCRIPTION
**Pull request by luciagirasoles**
#1177

### CODE REVIEW

**Comment by vinistock** [comment]
The PR looks good. We still need the CLA though. If I'm not mistaken, the bot expects it to be signed with the same email used on GitHub.

**Comment by luciagirasoles** [comment]
> The PR looks good. We still need the CLA though. If I'm not mistaken, the bot expects it to be signed with the same email used on GitHub.

The another PR passed the CLA verification https://github.com/Shopify/vscode-ruby-lsp/pull/895

**Comment by vinistock** [comment]
The bot was stuck indeed. I re-ran it and now it's fine.

We'll need to satisfy Sorbet's typechecking to get CI passing. Can you please create a file `sorbet/rbi/shims/yjit.rbi` and then add these contents to it?

```ruby
class RubyVM::YJIT
  def enable; end
end
```

**Comment by luciagirasoles** [comment]
@vinistock I still have problems with the CLA: 

> Error: Although you appear to have the correct authorization credentials, the `Shopify` organization has an IP allow list enabled, and your IP address is not permitted to access this resource.

File: sorbet/rbi/shims/yjit.rbi [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# typed: true
+
+class RubyVM::YJIT
+  def enable; end
```
[the remark this answers is not in the document]
Comment (luciagirasoles) [reply sorbet/rbi/shims/yjit.rbi]: Done!

**Review by vinistock — APPROVED** [review]
Thank you for the contribution!

**Pull request merged**